src/components/GradBackground.tsx

```diff
@@ -1,14 +1,8 @@
 import { FC } from 'react'
-import gradBG from '../assets/gradBG.svg'
-
 
 const GradBackground: FC = ({ children }) => {
```

Comment:
```suggestion
const GradBackground = ({ children }): JSX.Element => {
```

Reply:
@0-vortex what's wrong with FC, whereas it's a type that is intentionally created for React Component?

Reply:
I don't have a lot of context on FC as I am new to typescript, but after quick search it seems as if FC is beginner friendly but can becoming limited as the app gets more complex. 

https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html"
https://github.com/facebook/create-react-app/pull/8177